docs/ops/monitoring/checkpoint_monitoring.md

```diff
@@ -61,7 +61,7 @@ Note that for failed checkpoints, metrics are updated on a best efforts basis an
 - **ID**: The ID of the triggered checkpoint. The IDs are incremented for each checkpoint, starting at 1.
 - **Status**: The current status of the checkpoint, which is either *In Progress* (<i aria-hidden="true" class="fa fa-circle-o-notch fa-spin fa-fw"/>), *Completed* (<i aria-hidden="true" class="fa fa-check"/>), or *Failed* (<i aria-hidden="true" class="fa fa-remove"/>). If the triggered checkpoint is a savepoint, you will see a <i aria-hidden="true" class="fa fa-floppy-o"/> symbol.
 - **Trigger Time**: The time when the checkpoint was triggered at the JobManager.
-- **Latest Acknowledgement**: The time when the latest acknowledgement for any subtask was received at the JobManager (or n/a if no acknowledgement received yet). For a failed checkpoint, this is the time from trigger timestamp to failure.
+- **Latest Acknowledgement**: The time when the latest acknowledgement for any subtask was received at the JobManager (or n/a if no acknowledgement received yet).
```

Comment:
Update the `.zh.md` version as well?

Reply:
I don't know Chinese so I created FLINK-21122 to update it according to the [contribution guide](https://flink.apache.org/contributing/contribute-documentation.html#chinese-documentation-translation).